builder-18.toml

```diff
@@ -8,36 +8,36 @@ version = "0.10.1"
 
 [[buildpacks]]
   id = "heroku/maven"
-  uri = "https://github.com/heroku/heroku-buildpack-java/releases/download/v68/heroku-java-cnb-v68.tgz"
+  uri = "docker://public.ecr.aws/r2f9u0w4/heroku-maven-buildpack:0.2.0"
```

Comment:
What's the `r2f9u0w4` bit?

Reply:
That's our ECR repository identifier. I've applied for a custom one `heroku-buildpacks`, but their manual approval process takes longer than expected. The old identifier shown here stays valid indefinitely.